sunpy/instr/tests/test_rhessi.py

```diff
@@ -63,8 +61,8 @@ def test_parse_obssum_dbase_file():
 
 def test_parse_observing_summary_dbase_file():
     """
-    Test that we get the observing summary dbase file with the content
-    we expect.
+    Test that we get the observing summary dbase file with the content we
```

Comment:
dbase -> database? Unless I'm ignorant and "dbase" means something specific here...